enterprise/frontend/src/metabase-enterprise/embedding_iframe_sdk_setup/components/SelectEmbedOptionsStep.tsx

```diff
@@ -0,0 +1,131 @@
+import { Card, Checkbox, Divider, Group, Stack, Text } from "metabase/ui";
+
+import { useSdkIframeEmbedSetupContext } from "../context";
+
+import { DebouncedColorPillPicker } from "./DebouncedColorPillPicker";
+
+export const SelectEmbedOptionsStep = () => {
+  const { experience, settings, updateSettings } =
+    useSdkIframeEmbedSetupContext();
+
+  const { theme } = settings;
+
+  const isQuestionOrDashboardEmbed =
+    settings.dashboardId || settings.questionId;
+
+  const isExplorationEmbed = settings.template === "exploration";
+
+  const updateColor = useCallback(
+    (nextColors: Partial<MetabaseColors>) => {
+      updateSettings({
+        theme: { ...theme, colors: { ...theme?.colors, ...nextColors } },
+      });
+    },
+    [theme, updateSettings],
+  );
+
+  const isDashboardOrInteractiveQuestion =
+    settings.dashboardId ||
+    (settings.questionId && settings.isDrillThroughEnabled);
+
+  return (
+    <Stack gap="md">
+      {(isQuestionOrDashboardEmbed || isExplorationEmbed) && (
+        <Card p="md">
+          <Text size="lg" fw="bold" mb="md">
+            {t`Behavior`}
+          </Text>
+          <Stack gap="md">
+            {isQuestionOrDashboardEmbed && (
+              <Checkbox
+                label={t`Allow users to drill through on data points`}
+                checked={settings.isDrillThroughEnabled ?? false}
+                onChange={(e) =>
+                  updateSettings({ isDrillThroughEnabled: e.target.checked })
+                }
+              />
+            )}
+
+            {isDashboardOrInteractiveQuestion && (
+              <Checkbox
+                label={t`Allow downloads`}
+                checked={settings.withDownloads ?? false}
+                onChange={(e) =>
+                  updateSettings({ withDownloads: e.target.checked })
+                }
+              />
+            )}
+
+            {isExplorationEmbed && (
+              <Checkbox
+                label={t`Allow users to save new questions`}
+                checked={settings.isSaveEnabled ?? false}
+                onChange={(e) =>
+                  updateSettings({ isSaveEnabled: e.target.checked })
+                }
+              />
+            )}
+          </Stack>
+        </Card>
+      )}
+
+      <Card p="md">
+        <Text size="lg" fw="bold" mb="lg">
+          {t`Appearance`}
+        </Text>
+
+        <Group align="start" gap="xl" mb="lg">
+          {getConfigurableThemeColors().map(({ key, name, defaultColor }) => (
+            <Stack gap="xs" align="start" key={key}>
+              <Text size="sm" fw="bold">
+                {name}
+              </Text>
+
+              <DebouncedColorPillPicker
+                initialValue={theme?.colors?.[key] ?? defaultColor}
+                onChange={(color) => updateColor({ [key]: color })}
+                debounceMs={300}
+              />
+            </Stack>
+          ))}
+        </Group>
+
+        {isDashboardOrInteractiveQuestion && (
```

Comment:
This also feels odd. Why couldn't I select to show or hide the title on static question?

Reply:
@WiNloSt Agree. This is just because `<StaticQuestion />` currrently does not accept a title. Once we have a standalone `<Question />` we can delete every `isDashboardOrInteractiveQuestion` condition we have here.